react-search-component/react-search-app/src/App.css

```diff
@@ -0,0 +1,35 @@
+main {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  height: 100vh;
+  justify-content: center;
+  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
+  font-weight: 400;
+  padding-top: 20px;
+}
+
+li {
+  text-align: left;
+}
+
+.input-field {
+  width: 50%;
+  border-radius: 10px;
+  border: 1px solid black;
+  height: 25px;
+  padding-left: 30px;
+  margin-left: 20px;
```

Comment:
I would use `rem` for padding and margin! Check out the [efffective css guide](https://lms.learningfuze.com/code-guides/Learning-Fuze/curriculum/effective-css) for a refresher!